Migration script/openIMIS migration v1.4.2 - 1.5.0.sql

```diff
@@ -1912,3 +1914,6 @@ BEGIN CATCH
      ROLLBACK  TRANSACTION;  
 END CATCH  
 
+-- OTC-144: Set isOffline status to 0 for insurees in database 
+UPDATE tblInsuree set isOffline=0;
```

Comment:
cannot you add a condition:
 WHERE (SELECT TOP (1) [OfflineCHF]
  FROM [tblIMISDefaults]) is Null

Reply:
@delcroip I added condition 
`WHERE (SELECT 1 FROM tblIMISDefaults where OfflineCHF = 1 And OfflineHF = 1) is null;`